src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,33 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todos: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todos }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todos;
+
+  return (
+    <article
+      data-id={todos.id}
```

Comment:
```suggestion
  const {
    id,
    title,
    completed,
    user,
  } = todos;

  return (
    <article
      data-id={id}
```
it is better to also destruct the id and use it as follows